godtools/App/Features/Dashboard/Presentation/Dashboard/DashboardView.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  DashboardView.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 10/6/22.
+//  Copyright © 2022 Cru. All rights reserved.
+//
+
+import SwiftUI
+
+struct DashboardView: View {
+    
+    @ObservedObject var viewModel: DashboardViewModel
+    
+    private static let marginMultiplier: CGFloat = 15/375
+    
+    init(viewModel: DashboardViewModel) {
+        self.viewModel = viewModel
+        
+        UITabBar.appearance().barTintColor = .white
```

Comment:
Yeah, this would likely go away with a custom tab.  It's unfortunate that there's not a better way to set those values right now.  I'm looking into how much work it will be to create a custom tab bar, because it's not ideal either to have to build one from scratch

Reply:
I have an idea for a custom view we could do involving a Horizontal ScrollView and LazyHStack.  It would definitely take some work, but it would get us similar to what we had before.  I'd leave out allowing a user to pan gesture between tabs.  But something like this might work.  I wouldn't expect this to be done in this sprint either.  I may have some time to look into prototyping this.
```
struct DashboardView: View {
 var body...
GeometryReader 
    ScrollView (.horizontal) 
        LazyHStack { lessons view, home view, all tools view  }
    CustomTabBarView
}
```

Reply:
Ok I was reading through [this tutorial](https://truongvankien.medium.com/custom-tabview-in-swiftui-e7c0bf5667ab) and I realized it may be possible to use `TabView` to present the pages, but not use the built-in tab bar part.  The tutorial isn't easy to follow, but that's essentially what they did-- used TabView, didn't add the `TabItem`s, and built their own custom tab bar view that interfaces with TabView.  This should make things a lot simpler if that worked!

Reply:
Hey @rachaelblue that looks like a good start to me.  It seems we will need to create a custom View for the bottom tab bar then it's just a matter of determining which View to manage the lessons, home, and all tools.  If we can use ```TabView``` that would be the quickest solution.  At some point I still might look into encapsulating those Views in a scrollable LazyHStack just to get the scroll effect when tabbing between views, however, that isn't pressing.

Reply:
@levieggertcru do you want me to try the TabView solution with custom bottom tab bar in GT-1834 next?  Or are you thinking that should still wait for a different sprint?

Think I got all the other changes for this PR cleaned up though.